crates/ide_assists/src/handlers/add_turbo_fish.rs

```diff
@@ -50,8 +52,27 @@ pub(crate) fn add_turbo_fish(acc: &mut Assists, ctx: &AssistContext) -> Option<(
     let generics = hir::GenericDef::Function(fun).params(ctx.sema.db);
     if generics.is_empty() {
         mark::hit!(add_turbo_fish_non_generic);
+        mark::hit!(add_type_ascription_non_generic);
```

Comment:
It doesn't make much sense to have more than one test for a single mark, one of these tests can be safely removed.